operator/src/test/java/oracle/kubernetes/operator/helpers/KubernetesTestSupport.java

```diff
@@ -801,7 +812,9 @@ Object listResources(String namespace, Integer limit, String cont, String fieldS
     }
 
     private Predicate<Object> withLabels(String[] labelSelectors) {
-      return o -> labelSelectors == null || hasLabels(getMetadata(o), labelSelectors);
+      return o -> (labelSelectors == null
```

Comment:
To check for a null or empty array, use ArrayUtils.isEmpty()

Reply:
done